File: cdp-port/python/playwright_screenshot.py
from playwright.sync_api import sync_playwright

with sync_playwright() as playwright:
    try:
        chromium = playwright.chromium
        browser = chromium.launch()
        page = browser.new_page()
        page.goto('https://www.naver.com/')
        page.screenshot(path="naver.png", type="png", full_page=True)
        page.close()
    except Exception as e:
        print('htmlToImageFail : %s' % (e))

File: node/playwright/playwright_screenshot.js
const playwright = require('playwright');

(async () => {
  try {
    const browser = await playwright.chromium.launch();
    const context = await browser.newContext();
    const page = await context.newPage();
    await page.goto('https://www.naver.com/');
    await page.screenshot({ path: 'naver.png', type: 'png', fullPage: true });
    await browser.close();
  } catch (error) {
    console.log('htmlToImageFail :', error);
  }
})();